web_widget_many2many_binary_preview/static/src/web_widget_many2many_binary_preview.xml

```diff
@@ -0,0 +1,19 @@
+<templates>
+    <t
+        t-name="Many2ManyBinaryField.attachment_preview"
+        t-inherit="web.Many2ManyBinaryField.attachment_preview"
+        owl="1"
+    >
+        <xpath expr="//div[hasclass('o_image_box')]/a/span" position="attributes">
+            <attribute name="t-if">!show_preview(file)</attribute>
+        </xpath>
+        <xpath expr="//div[hasclass('o_image_box')]/a/span" position="after">
+            <img
+                t-if="show_preview(file)"
+                t-att-src="getUrl(file.id)"
+                aria-label="Download"
+                download=""
+            />
```

Comment:
As you already mentioned, the image previews can be rather sizy as compared to the more thumbnaily ones shown in the chatter attachment widget. I've noticed that this this is because of the URL; while here, getUrl returns `web/image/XXX?download=true`, in the other widget, it returns `/web/image/XXX/1920x160` which returns the thumbnail. Maybe you can inject another getUrl and do the same here as well. I don't think multiple sizes need to be supported at the moment, but it could be a roadmap item to provide parameters for that.